### PR TITLE
feat(kit):  switch to angular `i18nPlural` pipe

### LINKED ISSUE #258: [FEATURE] Angular built-in Pluralize
# 🚀 Feature request

### Is your feature request related to a problem?

We should deprecate our internal Russian-only pluralize implementation and switch to Angular built-in solution:
https://angular.io/api/common/I18nPluralPipe

### PR DESCRIPTION
**Pull request by vladimirpotekhin**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [ ] Bugfix
-   [x] Feature
-   [x] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #258

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/demo/src/modules/components/input-slider/input-slider.component.ts [hunk 1]
```diff
@@ -61,10 +61,16 @@ export class ExampleTuiInputSliderComponent extends AbstractExampleTuiControl {
...
-    readonly pluralizeVariants: ReadonlyArray<TuiPluralize> = [
+    readonly pluralizeVariants: ReadonlyArray<TuiPluralize | Record<string, string>> = [
         ['year', 'years', 'years'],
         ['thing', 'things', 'things'],
         ['year', 'years', 'years'],
+        {
+            '=0': 'zero',
+            '=1': 'one',
+            '=2': 'two',
+            other: 'other',
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/input-slider/input-slider.component.ts:72]: Let`s rewrite old examples (with leaving only one deprecated options + without new option) ?
```
readonly pluralizeVariants: ReadonlyArray<TuiPluralize | Record<string, string>> = [
        ['year', 'years', 'years'],
        {'=0': 'thing', '=1': 'things', '=2': 'things', other: 'things'}
        {'=0': 'year', '=1': 'years', '=2': 'years', other: 'years'}
]
```

File: projects/kit/abstract/slider/slider.ts [hunk 2]
```diff
@@ -136,20 +145,16 @@ export abstract class AbstractTuiSlider<T>
     ngOnInit() {
         super.ngOnInit();
 
-        const mouseMoves$ = typedFromEvent(this.documentRef, 'mousemove');
-        const mouseUps$ = typedFromEvent(this.documentRef, 'mouseup');
-        const touchMoves$ = typedFromEvent(this.documentRef, 'touchmove');
-        const touchEnds$ = typedFromEvent(this.documentRef, 'touchend');
+        const pointerMoves$ = typedFromEvent(this.documentRef, 'pointermove');
+        const pointerUps$ = typedFromEvent(this.documentRef, 'pointerup');
```
Comment (nsbarsukov) [inline projects/kit/abstract/slider/slider.ts:149]: 👍

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/components/input-slider/input-slider.component.ts (hunk 1 above)
Comment (vladimirpotekhin) [reply projects/demo/src/modules/components/input-slider/input-slider.component.ts]: Good idea, thanks!

File: projects/kit/abstract/slider/slider.ts (hunk 2 above)
Comment (vladimirpotekhin) [reply projects/kit/abstract/slider/slider.ts:149]: :sunglasses:

File: projects/kit/abstract/slider/slider.common.template.html [hunk 3]
```diff
@@ -1,8 +1,7 @@
 <div
     class="event-catcher"
     (tuiActiveZoneChange)="onActiveZone($event)"
-    (mousedown)="onMouseDown($event)"
-    (touchstart)="onTouchStart($event)"
+    (pointerdown)="onPointerDown($event)"
```
Comment (waterplea) [inline projects/kit/abstract/slider/slider.common.template.html:4]: :pepecry:
image

**Review by waterplea — APPROVED** [review]

**Pull request merged**